### PR TITLE
Json Processing 2.0 specification milestone draft

### PR DESCRIPTION
**Pull request by m0mus**
Signed-off-by: Dmitry Kornilov <dmitry.kornilov@oracle.com>

## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [ ] Specification PDF in the form of wombat-spec-x.y.pdf
- [ ] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on
      a Compatible Implementation and TCK, to ensure that the spec is
      implementable and testable.
- [ ] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://repo1.maven.org/maven2/jakarta/json/jakarta.json-api/2.0.0-RC2
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/master/nightly/jsonp-tck-2.0.0.zip
- [ ] The URL of the compatibility certification request issue:
      <add URL here>

Include the following in PR#2:
- [ ] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

File: jsonp/2.0/_index.md [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
+date: 2020-05-31
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta JSON Processing defines a Java(R) based framework for parsing, generating, transforming, and
 querying JSON documents.
 
-* [Jakarta JSON Processing 2.0 Release Record]()
+* [Jakarta JSON Processing 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jsonp/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta JSON Processing 2.0 Specification Document]() (PDF)
```
Comment (kwsutter) [inline jsonp/2.0/_index.md:11]: You have no plans to have a Spec available for Milestone 1?  That's okay.  But, I wouldn't remove these lines from the _index.md file.  I'd leave them with a blank url as they are.

File: jsonp/2.0/_index.md [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
+  * [jakarta.jsonp:jakarta.jsonp-api:jar:2.0.0-RC2](https://repo1.maven.org/maven2/jakarta/json/jakarta.json-api/2.0.0-RC2/jakarta.json-api-2.0.0-RC2.jar)
 
 # Compatible Implementations
 
-* [Jakarta JSON Processing]()
+* [Jakarta JSON Processing](https://repo1.maven.org/maven2/org/glassfish/jakarta.json/2.0.0-RC2/jakarta.json-2.0.0-RC2.jar)
 
 # Ballots
 
+## Release Review
```
Comment (kwsutter) [inline jsonp/2.0/_index.md:22]: You already have the "Release Review" section later in this _index.md file, there is no need to repeat it.  And, it should be after the Plan Review section anyway (where it currently exists).

File: jsonp/2.0/_index.md (hunk 1 above)
Comment (m0mus) [reply jsonp/2.0/_index.md:11]: JSONP doesn't have a spec document. Historically, the spec is just javadocs. Javadocs are jakartified. I'll remove these lines.

Comment (kwsutter) [reply jsonp/2.0/_index.md:11]: Thanks, @m0mus.  Didn't realize that jsonp relied solely on javadoc.

**Pull request merged**